apps/prairielearn/python/zygote.py

```diff
@@ -30,6 +30,7 @@
 
 import question_phases
 import zygote_utils as zu
+from faker import Faker
```

Comment:
It looks like this will add about 13MB of memory to the zygotes and their forked processes:

```python
>>> import psutil
>>> process = psutil.Process()
>>> print(process.memory_info().rss)
15269888
>>> from faker import Faker
>>> print(process.memory_info().rss)
28278784
```

I think this is acceptable, but there's always the option of using `importlib` to hook into the module loading machinery to seed the library just after it's been imported.

Reply:
I haven't done it in the autograder, is it worth doing it there?

Reply:
No, I don't think so.